src/segmantic/commands/monai_unet_cli.py

```diff
@@ -83,6 +83,40 @@ def train_config(
     monai_unet.train(**args)
 
 
+@app.command()
+def cross_validate(
+    config_file: Path = typer.Option(
+        ..., "--config-file", "-c", help="config file in json format"
+    ),
+    print_defaults: bool = False,
+):
+    """Train UNet with configuration provided as json file
+    Example invocation:
+        --config-file my_config.json
+    To generate a default config:
+        --config-file my_config.json --print-defaults
+    """
+    sig = inspect.signature(monai_unet.cross_validate)
+
+    if print_defaults:
+        default_args = {
+            k: v.default
+            if v.default is not inspect.Parameter.empty
+            else f"<required option: {v.annotation.__name__}>"
+            for k, v in sig.parameters.items()
+        }
+        config_file.write_text(json.dumps(default_args, indent=4))
```

Comment:
have a look at how the yaml is written, maybe the same code can be generalized to use for both functions instead of copy-pasting (an older version).